README.md

```diff
@@ -22,26 +22,45 @@ The hydra crawler is one of the components of the architecture. It will check if
 
 ## Dependencies
 
+### System
+
 This project uses `libmagic`, which needs to be installed on your system, eg:
+`brew install libmagic` on MacOS, or `sudo apt-get install libmagic-dev` on Linux.
+
+### Python
+
+This project uses Python 3.9.
```

Comment:
Nit: I believe i used python 3.12 at least for a few reviews and for the couple of PRs I created, and it seemed to work correctly. Would we want something like:

```suggestion
This project uses Python >=3.9.
```